tests/FileLockTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+$fname = 'tests/lock-test.txt';
+
+//create file
+$handle = fopen($fname, 'w') or die('ERROR[0]: Unable to open/create file "'.$fname.'"!');
+$txt = date('d.m.Y H:i:s');
+fwrite($handle, $txt);
+fclose($handle); 
+echo PHP_EOL .'Test file "'.$fname.'" created ['.date('d.m.Y H:i:s').']. FileSize: '.filesize($fname).' bytes.'. PHP_EOL . PHP_EOL; 
+
+
+class FileLockTest extends TestCase {
+
+	public function testCanBeLocked() {
+
+		global $fname;
```

Comment:
You should have a VERY big and good reason to use `global` keyword. And I don't see _any_ reason to use it here.

Reply:
agreed